lms/Jenkinsfile-tasks

```
@@ -24,6 +24,19 @@ def postSlack(state, env, task, region) {
     slackSend color: colors[state], message: messages[state]
 }
 
+def aws_regions = ['us-west-1', 'ca-central-1']
+
+def process_all_regions(environment, timeout, cmd, aws_regions_list) {
```

```suggestion
def process_regions(environment, timeout, cmd, aws_regions_list) {
```

Since the regions to use are a parameter, it might not be "all the regions" but some subset.

---

Sensible suggestion. I will provide an update.